modoboa_amavis/management/commands/amnotify.py

```diff
@@ -48,10 +48,13 @@ def _build_message(self, rcpt, total, reqs):
         if self.options["verbose"]:
             print("Sending notification to %s" % rcpt)
         content = render_to_string(
-            "modoboa_amavis/notifications/pending_requests.html", dict(
-                total=total, requests=reqs,
-                baseurl=self.baseurl, listingurl=self.listingurl
-            )
+            "modoboa_amavis/notifications/pending_requests.html",
+            {
+                'total': total,
+                'requests': reqs,
+                'baseurl': self.baseurl,
+                'listingurl': self.listingurl
+            }
```

Comment:
Actually, I think it would be better to build the context before, store it into a variable and replace this line part.